scripts/deploy.sh

```diff
@@ -12,7 +12,7 @@ src=`dirname $0`/../..
 
 # run unit tests
 cd $src/oauth-dropins && source local/bin/activate
-python -m unittest discover
+python -m unittest discover --pattern="test_*.py"
```

Comment:
oh yep, for me; prevents it from trying to run testutil.py as a unit test.

I kinda forgot I snuck that in there. :stuck_out_tongue: